target/linux/ramips/dts/mt7620a_engenius_esr600.dts

```diff
@@ -0,0 +1,211 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "engenius,esr600", "ralink,mt7620a-soc";
+	model = "EnGenius ESR600";
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x4000000>;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	leds {
+		// FIXME: locate WPS 5g blue LED
+		// ISSUE: taking the WAN LED on gpio2/5 
+		// causes AR8327 switch to fail initialization
+
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "esr600:amber:power";
+			gpios = <&gpio0 9 GPIO_ACTIVE_LOW>;
+		};
+		wps2g {
+			label = "esr600:amber:wps2g";
+			gpios = <&gpio2 6 GPIO_ACTIVE_LOW>;
+		};
+		wlan5g {
+			label = "esr600:blue:wlan5g";
+			gpios = <&gpio0 7 GPIO_ACTIVE_LOW>;
+		};
+		wlan2g {
+			label = "esr600:blue:wlan2g";
+			gpios = <&gpio3 0 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 1 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+			debounce-interval = <60>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio0 2 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+			debounce-interval = <60>;
+		};
+
+	};
+
+ };
+
+&gpio1 {
+	status = "okay";
+};
```

Comment:
Separate the nodes by newline here as well.

Reply:
OK.  Is there a suggested format for DTS files document somewhere that I missed?